force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -6040,6 +6040,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Frequency processed text</shortDescription>
         <value>Every {0} weeks</value>
     </labels>
+    <labels>
+        <fullName>RD_frequencyFirstAndFifteenth</fullName>
```

Comment:
I see you changed this _one_ label, but what about all the other `frequency*` labels above and below it? Consistency in labels in important. None of these should have been created without the RD prefix. Plus, these labels, which now all start with RD, should be moved in the label file to the RD section where they belong.

Reply:
prefixes were added and labels were relocated according to their category!